test/integration/conftest.py

```diff
@@ -20,12 +20,35 @@ def tmp_dir_and_file(ee_contents):
     return tmp_dir_and_file
 
 
-@pytest.fixture()
-def ee_tag(request, cli, container_runtime):
+def run(args, *a, **kw):
```

Comment:
@kdelee so... I re-organized a lot of your stuff.

The alternative here would be to introduce a module `test.integration.lib` and import from that. This is the only case where the function / class shenanigans were used in full force. Elsewhere I did it by moving the scope of a few things up to session.

I learned from the masters.